src/olympia/devhub/tests/test_views_subscription.py

```diff
@@ -76,9 +76,8 @@ def test_get(self, create_mock):
         ]
 
         # Get the page.
-        queries = 31
+        queries = 36
```

Comment:
I don't quite get why we have some many queries.. It seems we're fetching `previews`, `versions`, `files`, etc. many times for nothing.

Reply:
`previews`, `versions`, `files` etc come for the `transformer` that is called automatically when you fetch one or more add-ons. It's a relic from ancient times that is still quite useful in some places, and completely wasteful in others, so we've been scared to touch it.

That being said, this should happen *once*. If it happens multiple times, it means somewhere you're re-fetching an instance instead of re-using one, which is probably worth looking into.

Reply:
This is just an unknowledgeable drive-by, but I seem to recall seeing code that allows us to avoid using the transformer if we don't need it. Is that not possible in this case?

Reply:
We have a `no_transforms()` method, yes. But before adding that, we should figure out why there are duplicate queries for the add-on here.

Reply:
I tried and couldn't see anything obvious..

Reply:
I suspect this is not tied to your code at all and just a widespread problem in devhub. I'll take a look.

Reply:
The duplicates are caused by the my add-ons menu (that list all your add-ons) and some extra ownership checks in templates for the most part - We can deal with that in a follow-up, it affects all devhub, not just this page.